rubygems.js

```diff
@@ -28,3 +28,19 @@ export async function rubygemsUpdate(rubygemsVersionInput, rubyPrefix) {
 
   return true
 }
+
+async function rubygemsLatest(gem, platform, engine, version) {
```

Comment:
Could you add a link here to https://github.com/rubygems/rubygems/issues/7329 or another appropriate issue explaining why we need to workaround this bug in RubyGems?

Reply:
As mentioned, this has always been an issue, but since it's only tested here on Ruby 2.6, it failed because of the recently changed minimum Ruby version in RubyGems.

RubyGems 3.4 supported Ruby 2.6 and later, RubyGems 3.5 supports Ruby 3.0 and later.